frontend/src/scenes/experiments/Experiment.tsx

```diff
@@ -688,6 +715,16 @@ export function Experiment_(): JSX.Element {
                                                                 </Row>
                                                             </b>{' '}
                                                             {countDataForVariant(variant)}{' '}
+                                                            {areTrendResultsConfusing && idx === 0 && (
+                                                                <Tooltip
+                                                                    placement="right"
+                                                                    title="It might seem confusing that the best variant has lower absolute count, but this can happen when fewer people are exposed to this variant, so it's relative count is higher."
```

Comment:
```suggestion
                                                                    title="It might seem confusing that the best variant has lower absolute count, but this can happen when fewer people are exposed to this variant, so its relative count is higher."
```